.changeset/lemon-phones-sort.md

```diff
@@ -2,8 +2,21 @@
 "@vercel/edge-config": major
 ---
 
-- Use stale-while-revalidate during development
-- Don't throw generic errors
-- Make the return values read only
-- Use the `privateEdgeConfig` global when available
-- Export a clone function to support the read only behaviour
+ - **Return values are now read-only to improve in-memory caching**
+   In addition to the read-only change, the `@vercel/edge-config` package now also exports a `clone` function that can be used to make the read-only values modifiable again. Applications that used to modify the return value must now clone the value before doing the required changes.
```

Comment:
An intro paragraph with an example would be good. then the "in addition to.." is anchor in something more specific.

> Before this change it was possible to modify the values returned by the Edge Config SDK's methods. For example this was possible
>
> ```
> const someValue = await get("someKey")
> someValue.foo = 1
> ```
> Going forward any modifications will result in an error as the values returned by the SDK are now read-only.